admin-perspective/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
+
+## Critical Alerts
+![](img/notifications-open.png)
+- As this drawer is an endopoint for alertmanager, silenced and pending alerts do not appear here.
+- Clicking on an alert takes the user to the details page for the alert.
+- The alertmanager config could potentially be set up to send non-critical alerts here, but by default that would not be the case.
+- The critical alerts group should be expanded by default  and all others should be collapsed.
+
+## Available Updates
```

Comment:
This probably should just be an alert anyway.  We need to discuss what the design is, but info level alerts (not warning or critical) should be shown, and you should be able to silence them for a while with a reason just like any other alert.

Visually, I would suggest only reporting "there is an update available" vs showing the list.  I don't think we have evidence users select any other update than the latest.

Reply:
Is this section only available to cluster admins, aka specific roles?  Is the heading & section not displayed if you don't have access to update anything?